css.md

```diff
@@ -23,17 +23,29 @@ such that it's highly readable and consistent across different developers on a t
 - Put spaces after `:` in property declarations
 - Put spaces before `{` in rule declarations
 - Place `{` on the same line as the (last) selector and `}` on its own line
-- 
 
 ## Property Order
 
-Box model properties go first, in order from inside the box out, followed by positioning.
+Declare properties in three groups:
 
-Remaining properties are in alphabetical order.
+1. Display & box model
+   1. Switching `display` and/or `box-sizing`
```

Comment:
Yes, that's what I meant. You're right, it could use clarification. I'll
update it.